dtfs-central-api/src/services/portal/deal.service.update-status.test.ts

```diff
@@ -17,44 +17,46 @@ const dealId = 'dealId';
 const newStatus = DEAL_STATUS.CANCELLED;
 const auditDetails = generateSystemAuditDetails();
 
-describe('PortalDealService', () => {
+describe('PortalDealService - updateStatus', () => {
   beforeEach(() => {
     jest.clearAllMocks();
   });
 
-  describe('updateStatus', () => {
-    it(`calls updateGefDealStatus when submissionType is ${DEAL_TYPE.GEF}`, async () => {
-      // Arrange
-      const dealType = DEAL_TYPE.GEF;
-
-      // Act
-      await PortalDealService.updateStatus({ dealId, newStatus, auditDetails, dealType });
-
-      // Assert
-      expect(updateGefDealStatusMock).toHaveBeenCalledTimes(1);
-      expect(updateGefDealStatusMock).toHaveBeenCalledWith({
-        dealId,
-        newStatus,
-        auditDetails,
-      });
-      expect(updateBssEwcsDealStatusMock).toHaveBeenCalledTimes(0);
+  it(`calls updateGefDealStatus when submissionType is ${DEAL_TYPE.GEF}`, async () => {
```

Comment:
should call...

Reply:
@Zainzzkk aligned all test descriptions in this file.